### PR TITLE
Refactor and clarify event status and start/end date verification

### LINKED ISSUE #189: Remove unneeded values from EventStatus enumerated type to clarify the real-time intent of a WZDx feed
## Background
This issue was "discovered" following discussion about how to represent "recurrent events" in a WZDx feed. Based on several rounds of discussion culminating in a focused meeting on recurrent events in late July 2021, the group came to the consensus that a "recurrent event", that is, a work zone that occurs in a similar fashion multiple days or on a fixed schedule, should be represented with a unique WZDx road event for each occurrence of work. The start and end date time of each road event should reflect the start/end of that occurrence of work. 

In addition, recent discussion with many new members has shown that often individuals don't find WZDx's use case clearly apparent. It is another to do item to update the main project description to clarify the use of a WZDx feed is for making current or upcoming, real time (or as "real time" as possible), work zone information available to the public.

## Proposal
Because of the above discussions, the [EventStatus](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/EventStatus.md) enumerated type was examined. We determined three of the values—`pending`, `completed`, and `canceled`—should likely be removed. The reasons are as follows:

- `completed` has no utility because completed road events/work zones should not be included in a WZDx feed—a WZDx feed should include real-time information for the traveling public about current and planned road events
- `canceled` does not have utility becuase if a work zone road event is known to be canceled and will no longer occur, it should not be included in a WZDx feed.
- `pending` has a similar functionality to `planned`—they are describing the same concept but the distinction between both is not needed and the descriptions are unnecessarily precise. 

### EventStatus enumerated type after the above changes
Value | Description
--- | ---
`planned` | Indicates the road event is planned to occur in the future. All road events with a start date in the future should have this status. A road event with a date time range encompassing the current date and an `event_status` of `planned` indicates to the consumer that there is likely less accuracy in the road event data or that the work represented by the event hasn't begun yet.
`active` | Indicates that the road event is currently in place and work activity is occurring. It does not signify that there are workers present (see `workers_present` property on the [RoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/RoadEvent.md) for that information).

### PR DESCRIPTION
**Pull request by j-d-b**
This PR resolves #189. Specifically, it proposes the following changes:

- Deprecate the `event_status` property on the [WorkZoneRoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/WorkZoneRoadEvent.md) and [DetourRoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/DetourRoadEvent.md).
- Add deprecation note to [EventStatus](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/EventStatus.md) enumerated type.
- Deprecate the `start_date_accuracy` and `end_date_accuracy` properties on the WorkZoneRoadEvent and DetourRoadEvent.
- Add deprecation note to [TimeVerification](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/TimeVerification.md) enumerated type.
- Add new boolean properties `is_start_date_verified` and `is_end_date_verified` to the WorkZoneRoadEvent and DetourRoadEvent to indicate if the work zone or detour has been confirmed to have started/ended by a person or device in the field or report from a TMC, or similar.
- Require either `start_date_accuracy` or `is_start_date_verified`.
- Require either`end_date_accuracy` or `is_end_date_verified`.
- Update example feeds to use `is_start_date_verified` and `is_end_date_verified` and not use `event_status`, `start_date_accuracy`, or `end_date_accuracy`.

See #281 for a similar approach of using booleans that this PR is consistent with.

### CODE REVIEW

File: schemas/4.1/RoadEventFeature.json [hunk 1]
```diff
@@ -107,14 +107,13 @@
               "type": "string",
               "format": "date-time"
             },
-            "start_date_accuracy": {
-              "$ref": "#/definitions/TimeVerification"
+            "is_start_date_verified": {
+              "description": "Indicates if work has been confirmed to have started, such as from a person or device in the field or a report from a traffic management center",
```
Comment (j-d-b) [inline schemas/4.1/RoadEventFeature.json:111]: > has been confirmed to have started

This sounds a bit wordy to me, consider revising for clarity.

**Review by mollybehan — APPROVED** [review]

**Comment by sknick-iastate** [comment]
@j-d-b I thought we discussed at one point but were we planning on still updating the object and enumeration lists in the [spec-content README](https://github.com/usdot-jpo-ode/wzdx/tree/refactor/event-status/spec-content)?  Or will we just do this when everything is merged?

**Review by sknick-iastate — APPROVED** [review]

File: schemas/4.1/RoadEventFeature.json (hunk 1 above)
Comment (jlarsonOmahaNE) [reply schemas/4.1/RoadEventFeature.json:111]: I am in agreement. What about; 
"Indicates if work has been confirmed to have started, such as from a person, device or a report"

**Review by jlarsonOmahaNE — APPROVED** [review]

File: schemas/4.1/RoadEventFeature.json (hunk 1 above)
Comment (mark-mockett) [reply schemas/4.1/RoadEventFeature.json:111]: Co-chairs settled on "indicates if work has been confirmed to have started, such as from a person or field device."
